tests/next_tests/integration_tests/multi_feature_tests/ffront_tests/test_laplacian.py

```diff
@@ -86,3 +92,33 @@ def test_ffront_lap(cartesian_case):
         inout=out_field[2:-2, 2:-2],
         ref=lap_ref(lap_ref(in_field.array_ns.asarray(in_field.ndarray))),
     )
+
+
+def test_sdfgConvertible_laplap(cartesian_case):
+    in_field = cases.allocate(cartesian_case, laplap_program, "in_field")()
+    out_field = cases.allocate(cartesian_case, laplap_program, "out_field")()
+
+    def jit(executor):
+        def decorator(func):
+            def wrapper(*args, **kwargs):
+                if executor == run_dace_cpu:
+                    return dace.program()(func)(*args, **kwargs)
```

Comment:
I would prefer to assume that this test is only executed when the backend is run_dace_cpu, so you can decorate the program directly with `@dace.program`.

So you could put this line above the test function:
`@pytest.mark.skipif(cartesian_case.executor != run_dace_cpu)`